main.cpp

```diff
@@ -1,157 +1,167 @@
+        for example: 
+            noun: lightBulb.
+            action: turns on.  
+                This is a bad action because someone else turns on the light.  the light doesn't turn itself on. 
+            a better action would be: 
+                illuminates room
+                This is a better action, because this is what lightbulbs do.  
+            other actions that would work:
+                action: consume electricity
+                action: burn out and destroy filament.
+
+        
+ 3) write out how you'd call that action in pseudo code, in the space after the plain-english action
+ 
+ 4) If the action requires multiple words, use camelCaseToNameIt
+     don't forget the semi-colon after each statement
+ */
+
+
+//  example)
+//  Noun:       arm                     // 1)
+//  action 1:   the arm extends         // 2)
+arm.extend();                           // 3)
+//  action 2:   the arm flexes          // 2)
+arm.flex();                             // 3)
+//  action 3:   the arm rotates conter-clockwise
+arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
+
+
+//  1)
+//  Noun: car
+//  action 1: car moves Forward
+car.moveForward();
+//  action 2: car ignition turns on
+car.turnIgnitionOn();
+//  action 3: car headlights turn off
+car.turnHeadlightsOff();
+
+//  2)
+//  Noun: ship
+//  action 1: ship sounds honk
+ship.honk();
+//  action 2: ship raises flag
+ship.raiseFlag();
+//  action 3: ship takes anchor away
+ship.takeAnchorAway();
+
+//  3)
+//  Noun: window
+//  action 1: window breaks
+window.breaks();
+//  action 2: window goes dark
+window.goesDark();
+//  action 3: window closes vertically
+window.closeVertically();
+
+//  4)
+//  Noun: chair
+//  action 1: chair holds on two feet
+chair.holdsOnTwoFeet();
+//  action 2: chair makes weird noises
+chair.makesNoise();
+//  action 3: chair falls backwards
+chair.fallsBackwards();
+
+//  5)
+//  Noun: hand
+//  action 1: hand turns red
+hand.turnsRed();
+//  action 2: hand dissapears
+hand.dissapears();
+//  action 3: hand makes fist
+hand.makesFist();
+
+//  6)
+//  Noun: keyboard
+//  action 1: keyboard LED lights on
+keyboard.turnsLightsOn();
+//  action 2: keyboard ESC key turns off
+keyboard.turnsEscOff();
+//  action 3: keyboard touch turns ON
+keyboard.turnsTouchOn();
+
+//  7)
+//  Noun: guitar (Electric)
+//  action 1: guitar color changes to red
+guitar.changesColorToRed();
+//  action 2: guitar play E chord
+guitar.playsEChord();
+//  action 3: guitar play muted E Chord
+guitar.playsMutedEChord();
+
+//  8)
+//  Noun: pen
+//  action 1: pen make button stuck 
+pen.makesButtonStuck();
+//  action 2: pen add logo on top 
+pen.addsLogoOnTop();
+//  action 3: pen color turns black
+pen.changesColorBlack();
+
+//  9)
+//  Noun: speaker
+//  action 1: speaker power off
+speaker.powersOff();
+//  action 2: speaker change LED to blue
+speaker.changesLedColorBlue();
+//  action 3: speaker double its size
+speaker.doublesInSize();
```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.